src/js/login/containers/Main.jsx

```diff
@@ -32,14 +55,18 @@ export class Main extends React.Component {
     if (event.data === sessionStorage.userToken) { this.props.getUserData(); }
   }
 
+  getRedirectUrl = () => (new URLSearchParams(window.location.search)).get('next');
```

Comment:
URLSearchParams isn't imported in this file and was removed from another file. That might be breaking things in IE? We could add it to our polyfills file.

Reply:
Forgot to include it in this file. But adding to polyfills makes sense.